src/overlays/actors/ovl_En_Okarina_Tag/z_en_okarina_tag.h

```diff
@@ -7,10 +7,20 @@ struct EnOkarinaTag;
 
 typedef void (*EnOkarinaTagActionFunc)(struct EnOkarinaTag*, PlayState*);
 
+#define ENOKARINATAG_GET_7F(thisx) (((thisx)->params & 0x7F))
+#define ENOKARINATAG_GET_780(thisx) (((thisx)->params >> 0x7) & 0xF)
+#define ENOKARINATAG_GET_F800(thisx) (((thisx)->params >> 0xB) & 0x1F)
+
 typedef struct EnOkarinaTag {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnOkarinaTagActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x14];
+    /* 0x148 */ s16 unk148;
+    /* 0x14A */ s16 unk14A;
+    /* 0x14C */ s32 unk14C;
+    /* 0x150 */ s8 unk150;
+    /* 0x151 */ char pad151[3];
```

Comment:
```suggestion
    /* 0x151 */ UNK_TYPE1 pad151[3];
```
If it actually is a pad, this project likes to mark it by specifying the type as UNK_TYPE1 (which is u8, UNK_TYPE is u32)